### PR TITLE
add .strider.yml config

### PR DESCRIPTION
**Pull request by phiros**

### CODE REVIEW

**Comment by phiros** [comment]
This adds a config file for strider. Previously the .travis.yml file was used.
However, travis complains as soon as it sees something other than
linux or osx in the operating system config.

@authmillenon , @OlegHahm or @thomaseichinger  
It would be nice if someone could quickly ACK this. Travis & strider cd results are irrelevant for this file 
(nothing in RIOT uses or interferes with this file; the config is only used by the new version of strider).

**Comment by OlegHahm** [comment]
Well, ACK then.

File: .strider.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+allowed_to_fail:
+    - darwin
+    - freebsd
+
+before_install:
+    - source ./dist/tools/pr_check/check_labels.sh
+    - test -z "$TRAVIS_PULL_REQUEST" || test "$BUILDTEST_MCU_GROUP" = "static-tests" || check_gh_label "Ready for CI build" || exit 1
+
+install:
+    - git config --global user.email "no@spam.com"
```
Comment (OlegHahm) [inline .strider.yml]: Maybe we can change this to something else.

Comment (OlegHahm) [inline .strider.yml]: see https://tools.ietf.org/html/rfc2606#page-2

**Pull request merged**